docs/astro/sql/operators/append.rst

```diff
@@ -0,0 +1,43 @@
+======================================
+append operator
+======================================
+
+.. _append_operator:
+
+When to use the ``append`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can we restrict the tilde(s) to the text length above? How does it appear? I mean to see do we see extra tildes below the text? And shouldn't those be equal to the text above?

Reply:
@pankajkoti It doesn't need to be equal but not less than the heading text. But yes this looks odd, I will change it.